lib/client/conntest/kube.go

```diff
@@ -250,6 +256,42 @@ func (s KubeConnectionTester) handleErrFromKube(ctx context.Context, clusterName
 			connDiag, err := s.appendDiagnosticTrace(ctx, connectionDiagnosticID, traceType, message, actionErr)
 			return connDiag, trace.Wrap(err)
 		}
+		// WARNING: Check compatibility between this error message in the current version of
```

Comment:
Who is this warning for and what is the call to action here? It's not very clear.

If this depends on some error message strings to be maintained in sync between different versions, can they be moved into constants? Otherwise it's not super clear what should be checked and when.